src/core/client.ts

```diff
@@ -171,10 +170,10 @@ export class OakConsentClient {
         appSlug: parsed.app,
       }));
 
-      return consents;
+      return [parsed.user, consents];
```

Comment:
Choosing to pull the id from the root of the cookie object to handle the case where consents are empty. I'm going to need that for ENG-795. There i'll have to start persisting the `userId` on initialisation (I.e before the user has submitted any consents)